src/config/utils.ts

```diff
@@ -0,0 +1,48 @@
+import { ETHEREUM_NETWORK } from '../logic/wallets/getWeb3'
```

Comment:
If we move to use the `src/config/index.ts` "service". We can remove this file as well.